src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -199,6 +204,20 @@ public virtual Publisher.IPublisherClient GrpcClient
             get { throw new NotImplementedException(); }
         }
 
+        /// <summary>
```

Comment:
It's great to see all this documentation. Will run docfx on it and update my github pages when it's merged.